diff -ur gamine-1.1.ORIG/gamine.c gamine-1.1.CHANGED/gamine.c
--- gamine-1.1.ORIG/gamine.c	2011-01-01 13:22:17.000000000 +0100
+++ gamine-1.1.CHANGED/gamine.c	2015-04-30 12:04:57.742886522 +0200
@@ -13,7 +13,6 @@
 #include <time.h>
 #include <gst/gst.h>
 #include <gdk/gdkkeysyms.h>
-#include <gconf/gconf-client.h>
 
 //gettext
 #include <libintl.h>
@@ -28,7 +27,6 @@
 typedef struct {
     GstBus *bus;
     GtkWidget *da;
-    GConfClient* gc;
     cairo_surface_t *surface;
     cairo_t *cr;
     gboolean *is_cairo;
@@ -43,15 +41,8 @@
 static void
 load_conf (gamine_t *cb)
 {
-    linewidth = gconf_client_get_int(cb->gc, "/apps/gamine/line_width", NULL);
-    if ( linewidth == 0 )
         linewidth = 10;
-    objectweight = gconf_client_get_int(cb->gc, "/apps/gamine/object_weight", 
-        NULL);
-    if ( objectweight == 0 )
         objectweight = 15;
-    fontweight = gconf_client_get_int(cb->gc, "/apps/gamine/font_weight", NULL);
-    if ( fontweight == 0 )
         fontweight = 50;
 }
 
@@ -329,7 +320,6 @@
     cb.is_cairo = FALSE;
     gtk_init (&argc, &argv);
     gst_init (&argc, &argv);
-    gconf_init(argc, argv, NULL);
     gst_play_background (cb.bus,
               "BachJSBrandenburgConcertNo2inFMajorBWV1047mvmt1.ogg", TRUE);
 
@@ -341,7 +331,6 @@
     bg_color.blue  = 65535;
     gtk_widget_modify_bg (cb.da, GTK_STATE_NORMAL, &bg_color);
     gtk_container_add (GTK_CONTAINER (window), cb.da);
-    cb.gc = gconf_client_get_default();
 
     gtkwindow = GTK_WINDOW(window);
     gtk_window_set_title (gtkwindow, "Gamine");
diff -ur gamine-1.1.ORIG/Makefile gamine-1.1.CHANGED/Makefile
--- gamine-1.1.ORIG/Makefile	2011-01-01 13:22:17.000000000 +0100
+++ gamine-1.1.CHANGED/Makefile	2015-04-30 12:06:06.156234137 +0200
@@ -3,19 +3,14 @@
 datadir = $(PREFIX)/share
 pkgdatadir = $(datadir)/gamine
 docdir = $(datadir)/doc/gamine
-sysconfdir = $(PREFIX)/etc
 desktopdir = $(datadir)/applications
 icondir = $(datadir)/icons/hicolor/scalable/apps
 localedir = $(datadir)/locale
 
-GCONFTOOL = /usr/bin/gconftool-2
-GCONF_SCHEMA_CONFIG_SOURCE = xml:merged:/etc/gconf/gconf.xml.defaults
-GCONF_SCHEMA_FILE_DIR = $(sysconfdir)/gconf/schemas
-GCONF_DISABLE_MAKEFILE_SCHEMA_INSTALL = 0
 CFLAGS = -Wall
 #CFLAGS = -Wall -g 
-CPPFLAGS = $(shell pkg-config --cflags gtk+-2.0 cairo glib-2.0 gstreamer-0.10 gconf-2.0)  -DDATADIR=\""$(pkgdatadir)"\"  -DLOCALDIR=\""$(localedir)"\"
-LDLIBS = $(shell pkg-config --libs gtk+-2.0 cairo glib-2.0 gstreamer-0.10 gconf-2.0)  -DDATADIR=\""$(pkgdatadir)"\"  -DLOCALDIR=\""$(localedir)"\"
+CPPFLAGS = $(shell pkg-config --cflags gtk+-2.0 cairo glib-2.0 gstreamer-0.10)  -DDATADIR=\""$(pkgdatadir)"\"  -DLOCALDIR=\""$(localedir)"\"
+LDLIBS = $(shell pkg-config --libs gtk+-2.0 cairo glib-2.0 gstreamer-0.10)  -DDATADIR=\""$(pkgdatadir)"\"  -DLOCALDIR=\""$(localedir)"\"
 LDFLAGS = -g 
 CC = gcc
 target = gamine
@@ -37,16 +32,13 @@
 	mkdir -p $(docdir)
 	mkdir -p $(icondir)
 	mkdir -p $(desktopdir)
-	mkdir -p $(GCONF_SCHEMA_FILE_DIR)
 	mkdir -p $(localedir)/fr/LC_MESSAGES
 	install -m 755 gamine $(bindir)/
 	install -m 644 pencil.png $(pkgdatadir)/
 	install -m 644 gamine.png $(pkgdatadir)/
 	install -m 644 sounds/* $(pkgdatadir)/sounds/
 	install -m 644 README.pencil README.sounds README ChangeLog COPYING $(docdir)/
-	install -m 644 gamine.schemas $(GCONF_SCHEMA_FILE_DIR)/
 	install -m 644 locale/fr.mo $(localedir)/fr/LC_MESSAGES/gamine.mo
-	GCONF_CONFIG_SOURCE=$(GCONF_SCHEMA_CONFIG_SOURCE) $(GCONFTOOL) --makefile-install-rule $(GCONF_SCHEMA_FILE_DIR)/gamine.schemas
 	install -m 644 gamine.desktop $(desktopdir)/
 	install -m 644 gamine.svg $(icondir)/
 
@@ -54,6 +46,5 @@
 	rm -rf $(bindir)/gamine
 	rm -rf $(pkgdatadir)
 	rm -rf $(docdir)
-	rm -rf $(GCONF_SCHEMA_FILE_DIR)/gamine.schemas
 	rm -rf $(desktopdir)/gamine.desktop
 	rm -rf $(icondir)/gamine.svg
